### PR TITLE
Sh/action.yml

### PR DESCRIPTION
**Pull request by sshjelle**
# Changes
- action.yml is tested and works
- add github/md2confluence.yml - tested and works
- add github/md2confluenceUsingAction.yml - tested and works

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    required: true
+  CONFLUENCE_SPACE_ID:
+    description: 'Confluence space ID'
+    required: true
+  CONFLUENCE_PARENT_PAGE_ID:
+    description: 'ID of the confluence parent page where the markdown files will be published under'
+    required: true
+  MARKDOWN_FOLDER:
+    description: 'Path to the folder containing the markdown files to be published. Set to root (./) if all markdown files in your repo should be published.'
+    required: true
```
Comment (ottolote) [inline action.yml]: could change this to false, default is working dir 👍 
```suggestion
    required: false
```

**Review by ottolote — DISMISSED** [review]
Looks good! 👍 

Putting the `github` folder in `docs/` might make sense but I'll leave it up to you

**Comment by sshjelle** [comment]
@ottolote, pushed the changes you suggested, need another approval before merge

**Review by ottolote — APPROVED** [review]
👍

**Pull request merged**